howso/utilities/tests/test_utilities.py

```diff
@@ -680,3 +686,12 @@ def test_matrix_processing_normalization_zero_division():
     }, index=['a', 'b', 'c']).T
 
     assert_frame_equal(processed_matrix, correct_matrix)
+
+
+def test_confusin_matrix_formating():
```

Comment:
Can this test by `pytest.parametrized`? Would be nice to test 

1. Empty dicts
2. improperly formatted dicts

Reply:
Python doesn't allow empty dicts, I'll add other tests